network/discovery/forking_dv5_listener.go

```diff
@@ -95,9 +95,5 @@ type annotatedIterator struct {
 }
 
 func (i *annotatedIterator) Next() bool {
-	if !i.Iterator.Next() {
-		return false
-	}
-	metricIterations.WithLabelValues(i.fork).Inc()
-	return true
+	return i.Iterator.Next()
 }
```

Comment:
stage branch previously had removed dual discovery, but it has since been reverted after we've encountered issues without it

so we still need a metric that's broken down to pre-fork and post-fork discoveries at the moment

the other metrics for discovery accepted/rejected can remain as they are, as they're at a higher level than this

the metric here was just measuring the performance of each iterator in order to compare pre-fork vs post-fork discoveries